### PR TITLE
SEO-176113-Image-Alt-Text-Typescript

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the image alt text.

Regards,
Beryl.

### CODE REVIEW

File: TypeScript/CheckBox/Easy-customization.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Easy-customization
-description: easy customization
+title: Easy-customization in TypeScript Checkbox | Syncfusion
```
Comment (Yvone-Atieno) [inline TypeScript/CheckBox/Easy-customization.md:3]: Add the word Control

File: TypeScript/FileExplorer/Accessing-Shared-Folder.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Accessing shared folder | FileExplorer | Javascript | Syncfusion
-description: Accessing shared folder option in FileExplorer
+title: Accessing shared folder in Typescript FileExplorer | Syncfusion
```
Comment (Yvone-Atieno) [inline TypeScript/FileExplorer/Accessing-Shared-Folder.md:2]: Add the word Control

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: TypeScript/CheckBox/Easy-customization.md (hunk 1 above)
Comment (berylouma) [reply TypeScript/CheckBox/Easy-customization.md:3]: Updated

File: TypeScript/FileExplorer/Accessing-Shared-Folder.md (hunk 2 above)
Comment (berylouma) [reply TypeScript/FileExplorer/Accessing-Shared-Folder.md:2]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**